reservation/views.py

```diff
@@ -29,15 +29,15 @@ def get_csv_file(request):
         writer = csv.writer(response)
         writer.writerow(['ID', 'User', 'Hall', 'Movie', 'Seat', 'Start time'])
         for r in reservations:
-
-            writer.writerow([r.pk,
-                             r.user,
-                             r.playing_time.assigned_hall,
-                             r.playing_time.assigned_movie,
-                             r.seat,
-                             r.playing_time.start_time,
-                             r.is_confirmed,
-                             ])
+            if r.is_confirmed:
```

Comment:
You could filter by this when you query the reservation objects to avoid the extra check here.
For me, it would be better to have all the reservations exported in the csv but have them somehow marked so that I could be warned that I should go and confirm them. Either put them on top or even have some unicode characters like “⚠” (U+26A0) along the unconfirmed ones.